[IAM] Updates to 2FA doc to explain account recovery due to 2FA issues (#12155)
cloudflare/cloudflare-docs: content/fundamentals/account-and-billing/account-security/2fa.md

## Recover your account

If you do not have access to your 2FA account or backup codes and cannot currently generate a 2FA code, use a verified device that you have logged in from before to request a temporary access code.

1. Log into the [Cloudflare dashboard](https://dash.cloudflare.com/login).

> 2. On the **Two-Factor Authentication** page, select **Try recovery** on **Lost all 2FA devices and backup codes?*.

Reviewer: ```suggestion
2. On the **Two-Factor Authentication** page, select **Try recovery** on **Lost all 2FA devices and backup codes?**.
```

3. Select **Begin recovery**.
4. A 6-digit access code will be sent to the email address associated with your Cloudflare account.
5. Enter the temporary access code into the Cloudflare Dashboard and select **Verify email**.
6. Select **Verify device**. This checks whether you are using a device that has previously logged into your account.
7. If you receive a "Device verification failed" message:

    - If you clear your cookies often or are logging in from a different IP address, you have wiped Cloudflare's memory of your device and will need to use a different device to verify.
   - For security reasons, you must wait three to five days after completing verification to receive your temporary access code.
    - Your browser may be set to clear cookies on exit or after browser or OS upgrades. This interferes with the device verification process.
    - You may be using anti-malware or other software that automatically clears your browser cookies and makes your device unregognizable by Cloudflare's Dashboard.

> If at this point you are still unable to successfully enter the Cloudflare Dashboard, [contact Support](/support/contacting-cloudflare-support/).

Reviewer: ```suggestion
If you are still unable to verify your device, [contact Support](/support/contacting-cloudflare-support/).
```

2. On the **Two-Factor Authentication** page, select **Try recovery** on **Lost all 2FA devices and backup codes?*.
3. Select **Begin recovery**.
4. A 6-digit access code will be sent to the email address associated with your Cloudflare account.
5. Enter the temporary access code into the Cloudflare Dashboard and select **Verify email**.
6. Select **Verify device**. This checks whether you are using a device that has previously logged into your account.
7. If you receive a "Device verification failed" message:

    - If you clear your cookies often or are logging in from a different IP address, you have wiped Cloudflare's memory of your device and will need to use a different device to verify.
   - For security reasons, you must wait three to five days after completing verification to receive your temporary access code.
    - Your browser may be set to clear cookies on exit or after browser or OS upgrades. This interferes with the device verification process.
    - You may be using anti-malware or other software that automatically clears your browser cookies and makes your device unregognizable by Cloudflare's Dashboard.

> 8. If at this point you are still unable to successfully enter the Cloudflare Dashboard due to a 2FA issue please **Contact Support**

Reviewer: ```suggestion
If at this point you are still unable to successfully enter the Cloudflare Dashboard, [contact Support](/support/contacting-cloudflare-support/).
```

> https://dash.cloudflare.com/login-help

Reviewer: ```suggestion
```

## Recover your account

If you do not have access to your 2FA account or backup codes and cannot currently generate a 2FA code, use a verified device that you have logged in from before to request a temporary access code.

1. Log into the [Cloudflare dashboard](https://dash.cloudflare.com/login).
2. On the **Two-Factor Authentication** page, select **Try recovery** on **Lost all 2FA devices and backup codes?*.
3. Select **Begin recovery**.
4. A 6-digit access code will be sent to the email address associated with your Cloudflare account.

> 5. Input the temporary access code into the Cloudflare Dashboard and click "Verify email" to verify the email address.

Reviewer: ```suggestion
5. Enter the temporary access code into the Cloudflare Dashboard and select **Verify email**.
```

> 6. You will now be asked to verify your device using a device that you have logged in from before by clicking on "Verify device"

Reviewer: ```suggestion
6. Select **Verify device**. This checks whether you are using a device that has previously logged into your account.
```

> 7. If you receive a "Device verification failed" message you should check the following

Reviewer: ```suggestion
7. If you receive a "Device verification failed" message:
```

>     1. If you clear your cookies often or are logging in from a different IP address, you have wiped Cloudflare's memory of your device and will         need to use a different device to verify.

Reviewer: ```suggestion
    - If you clear your cookies often or are logging in from a different IP address, you have wiped Cloudflare's memory of your device and will need to use a different device to verify.
```

>     2. For security reasons, you must wait three to five days after completing verification to receive your temporary access code, so please wait 5      days after successful verification before reaching out directly to Cloudflare for support with your 2FA issue

Reviewer: ```suggestion
   - For security reasons, you must wait three to five days after completing verification to receive your temporary access code.
```

>     3. Errors like "Device verification failed" are usually a results of missing cookies, so sure that you have your browser set to save cookies and     not to clear cookies on exit or after browser and OS Upgrades.

Reviewer: ```suggestion
    - Your browser may be set to clear cookies on exit or after browser or OS upgrades. This interferes with the device verification process.
```

>     4. Make sure that you don't have any other software such as anti-malware that may be automatically clearing your browser cookies and making your 

Reviewer: ```suggestion
    - You may be using anti-malware or other software that automatically clears your browser cookies and makes your device unregognizable by Cloudflare's Dashboard.
```

>     device unregognizable by Cloudflare's Dashboard

Reviewer: ```suggestion
```
